sdk/cosmos/azure-cosmos/azure/cosmos/aio/_container.py

```diff
@@ -517,7 +510,7 @@ def query_items_change_feed(
             partition_key: PartitionKeyType,
             priority: Optional[Literal["High", "Low"]] = None,
             mode: Optional[Literal["LatestVersion", "AllVersionsAndDeletes"]] = None,
-            response_hook: Optional[Callable[[Mapping[str, Any], AsyncItemPaged[Dict[str, Any]]], None]] = None,
+            response_hook: Optional[Callable[[Dict[str, str], Dict[str, Any]], None]] = None,
```

Comment:
Why type this is a `Dict`? We don't want people mutating it?

Reply:
yep you're right, will change back